README.md

```diff
@@ -66,4 +66,10 @@ yarn test
 
 Open `app/tests/index.html` in browser
 
-Check us out at https://www.cardanolite.com
\ No newline at end of file
+Check us out at https://www.cardanolite.com
+
+---
+Donations are really appreciated!
+[BTC](https://www.blockchain.com/btc/address/1DECAF2uSpFTP4L1fAHR8GCLrPqdwdLse9) /
```

Comment:
I would put the address in the link title - it would be then look like this:
BTC: [0x...](https://explorer.com/0x....)

This applies to ETH and ADA as well